src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
+  const styleObject = {};
+
+  const declarations = styles.split(';');
```

Comment:
[CODE PRACTICE] - Instead of using a `forEach` loop to build the `styleObject`, consider using more functional approaches like `reduce` which aligns with the checklist's recommendation to avoid loops and use iteration methods.